controller/lib/core/blower_pid.cpp

```diff
@@ -22,15 +22,11 @@ limitations under the License.
 #include "sensors.h"
 #include "types.h"
 
-static float Setpoint;
-static float Input;
-static float Output;
-
 // PID-tuning were chosen by following the Ziegler-Nichols method,
 // https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method
 //
 // Note that Ku and Tu only seem to work with this particular sample time.
-static constexpr Duration PID_SAMPLE_TIME = milliseconds(10);
+static constexpr Duration PID_SAMPLE_PERIOD = milliseconds(10);
```

Comment:
OK, I stopped reviewing when I got here because I see that this PR contains both your changes and also Eugene's PID changes. 

Can you rebase so it's just your changes?  sglow wrote https://github.com/RespiraWorks/VentilatorSoftware/wiki/Workflow-&-Processes#hints-on-rebasing-a-branch and I am also able to help interactively if necessary.

Reply:
Yes and no: I can roll back those changes quite easily, but I don't want to spend too much time updating the soon-to-be obsolete pid library (especially considering how simple it is on the new one!), and I will let it be inconsistent with the rest of this PR until we get the new library in master. Does that work for you?

Reply:
Ah, I didn't realize the PID changes hadn't been checked in!

In that case you just need to update the base branch here in github; change it from "master" to the name of the PID branch.  Then once the PID changes land, change it back to master.

(As it happens, this process is automated by a tool I recently wrote: https://github.com/jlebar/git-pr-chain)

Reply:
(The problem I am trying to solve is, when I review this PR, I want to see only the changes in this PR; I don't want to disambiguate which changes here came from the PID PR and which you're making, because I'm afraid I'll do it wrong, either miss something or make comments that aren't relevant to your changes.)

Reply:
I understand perfectly what your problem was.
Of course I didn't see your message while I was rebasing... it is quite fast though and shouldn't be to much trouble do rebase once more...

Reply:
Thanks.  :)